frontend/control-center/src/pages/Connectors/ConnectedChannelsList/DisableModal/index.tsx

```diff
@@ -51,8 +51,10 @@ export const DisableModal = (props: DisableModalProps) => {
             }}
             type="submit"
           >
-            {isLoading && <RefreshIcon height={24} width={24} />}
-            {isLoading ? t('unsubscribing') : error ? t('tryAgain') : t('confirm')}
+            <>
```

Comment:
Why is this necessary? 🤔

Reply:
we need the fragment to wrap several children

Reply:
Locally I can build this without the fragment. Do you get an error?

Reply:
You're right! I thought I had an error about this, but I actually tested it now, and it works without the fragment. Will fix now 👍